src/main/java/org/apache/sysml/runtime/matrix/data/LibMatrixDatagen.java

```diff
@@ -510,14 +511,19 @@ private static void genRandomNumbers(boolean invokedFromCP, int rl, int ru, int
 
 		// Setup Pseudo Random Number Generator for cell values based on 'pdf'.
 		if (valuePRNG == null) {
-			if ( rgen._pdf.equalsIgnoreCase(RAND_PDF_UNIFORM)) 
+			switch (rgen._pdf) {
+			case UNIFORM:
```

Comment:
I tried reformatting using the style format that @deroneriksson checked in (#515). It prefers this style - i.e. `case` at the same indentation level as `switch`.